examples/common.h

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2016 Svyatoslav Mishyn <juef@openmailbox.org>
+ *
+ * Permission to use, copy, modify, and/or distribute this software for
+ * any purpose with or without fee is hereby granted, provided that the
+ * above copyright notice and this permission notice appear in all
+ * copies.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL
+ * WARRANTIES WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE
+ * AUTHOR BE LIABLE FOR ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL
+ * DAMAGES OR ANY DAMAGES WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR
+ * PROFITS, WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER
+ * TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE USE OR
+ * PERFORMANCE OF THIS SOFTWARE.
+ */
+
+#ifndef COMMON_H
+#define COMMON_H
+
+#include <stddef.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include <curi.h>
+
+enum	exit_status {
+	EXIT_ERR = -2,
+	EXIT_OK  = -1
+};
+
+static inline const char *
```

Comment:
in `common.h` I used `static` because I followed first rule (A simple portable mode): <http://www.greenend.org.uk/rjk/tech/inline.html>;

For others, it is just a habit (if local function - should be `static`). They could be removed, as those functions are used in a single unit..

Should I remove `static` keyword from `decode_url.c` and `split_uri.c` for all functions ?